src/views/local-token-service.handlebars

```diff
@@ -123,6 +123,15 @@ let text = client.get("<span class="origin">https://gateway.local</span>/things"
         </div>
       </div>
     </section>
+    <script>
+/* postToken to client apps if any */
+if (config.get('oauthPostToken') && (window.location.protocol === "file:") {
```

Comment:
This is a handlebars template that gets used in the UI. Instead of using `config`, you'll need to pass a value into the template.

Reply:
yes that code didnt work actually I figured out I used other hack to get token, next version works anyway